tests/test_batch.py

```diff
@@ -0,0 +1,96 @@
+"""Test for batch computations.
+
+Only finufft and cufinufft support batch computations.
+"""
+import numpy as np
+import numpy.testing as npt
+from pytest_cases import parametrize_with_cases, parametrize, fixture
+import pytest
+
+from mrinufft import get_operator
+from mrinufft.operators.interfaces import CUFINUFFT_AVAILABLE
+from case_trajectories import CasesTrajectories
+
+
+@fixture(scope="module")
+@parametrize(
+    backend=[
+        "finufft",
+        pytest.param(
+            "cufinufft",
```

Comment:
I recommend this as a separate test, with a tag like `def test_gpu_<name>` 
That way, we can make a CI just to run this. 
Or we could also add an environment variable `ONLY_RUN_GPU` which can be enabled to skip all tests except the ones needing GPU. WDYT? 

That way the CPU tests can happen online while GPU ones can happen offline, giving faster CI run times

Reply:
instead of rewriting the test for gpu cases (that would create a lot of redundancy) we can use pytest custom markers[^1] and command line argument. 

[^1]: https://docs.pytest.org/en/7.1.x/example/markers.html